src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltype/InputIllegalTypeSameFileName.java

```diff
@@ -26,4 +26,7 @@ private void foo() {
     }
     java.util.List<Integer> list = new ArrayList<>(); //WARNING
     private ArrayList<String> values;
+    private Boolean d;
```

Comment:
please put "//WARNING" to lines where violation is expected (see examples above).
This file is used for certain configuration, so it should be clear by reading Input file where violations are.

Reply:
fixed